app/ante.go

```diff
@@ -57,6 +60,8 @@ func NewAnteHandler(
 		ante.NewSigGasConsumeDecorator(ak, sigGasConsumer),
 		ante.NewSigVerificationDecorator(ak, signModeHandler),
 		ante.NewIncrementSequenceDecorator(ak),
-		ibcante.NewAnteDecorator(channelKeeper),
+		// UNFORKINGTODO C: I think this is correct in changing NewAnteDecorator to NewRedundantRelayDecorator, but want ACK
+		//ibcante.NewAnteDecorator(channelKeeper),
+		ibcante.NewRedundantRelayDecorator(channelKeeper),
```

Comment:
ACK (this is also mempool only, so doesnt matter if were wrong)

Reply:
https://github.com/osmosis-labs/osmosis/pull/6413/commits/dbc2bde371fd6d077781084c33e319566bc55421